services/api/src/models/Quests.ts

```diff
@@ -128,7 +134,8 @@ function mailNewQuestToAdmin(mail: MailService, quest: Quest) {
     over ${quest.mintimeminutes} - ${quest.maxtimeminutes} minutes.
     ${quest.genre}.
     ${quest.requirespenpaper ? 'Requires pen and paper.' : 'No pen or paper required.'}
-    ${quest.expansionhorror ? 'Requires The Horror expansion.' : 'No expansions required.'}`;
+    ${quest.expansionhorror ? 'Requires The Horror expansion.' : 'No expansions required.'}
+    ${quest.expansionfuture ? 'Request The Future expansion.' : ''}`;
```

Comment:
Is this intentionally different?

Reply:
Was trying to be clever with the whole Future requiring Horror thing, but esp since this is an admin, simplified it to just say Horror y/n, Future y/n